### ISSUE TITLE
Adjust README

### ISSUE THREAD

**Issue by keilw** [eclipse-ee4j/jsonp#79]
The README file should no longer point to the "javaee" organization but this repository instead.

**Comment by ExE-Boss** [comment]
Yes, but right now, the website is still hosted [at the old location](https://javaee.github.io/jsonp/).

**Comment by keilw** [comment]
I know, I created it ;-) Already flagged that in more than just this project, but it seems to take a bit more time.

**Comment by m0mus** [comment]
@keilw web site has not been transferred yet. We will put a header on the old site indicating that it's related to Java EE and future development of JSON-P is moved to the Eclipse Foundation. It's in the backlog.

**Comment by keilw** [comment]
@m0mus So what is the purpose of the Wiki, that is not a full web site with Bootstrap, etc.? And are others in the project also able to contribute to that wiki? Congrats on the Star Spec Lead btw, having helped 2 JSON JSRs get done in EE8 you deserved it for 2017.

**Comment by mswatosh** [comment]
Readme was updated years ago to remove javaee references, closing this issue
